src/Symfony/Bundle/FrameworkBundle/Command/ContainerDebugCommand.php

```diff
@@ -106,7 +107,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $options = array('tag' => $tag, 'show_private' => $input->getOption('show-private'));
         } elseif ($name = $input->getArgument('name')) {
             $name = $this->findProperServiceName($input, $io, $object, $name);
-            $options = array('id' => $name);
+            $options = array('id' => $name, 'injections' => $this->findServiceIdsInjections($object, $name));
```

Comment:
should rather be named ``usages`` than ``injections``. this is where the service is used